solidity/dashboard/src/components/WithdrawETHModal.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useCallback } from "react"
+import { SubmitButton } from "./Button"
+import { useCustomOnSubmitFormik } from "../hooks/useCustomOnSubmitFormik"
+import FormInput from "./FormInput"
+import { getErrorsObj } from "../forms/common-validators"
+import { withFormik } from "formik"
+import { colors } from "../constants/colors"
+import web3Utils from "web3-utils"
+import { useWeb3Context } from "./WithWeb3Context"
+import { tbtcAuthorizationService } from "../services/tbtc-authorization.service"
+import { useShowMessage, messageType } from "./Message"
+import * as Icons from "./Icons"
+
+const AvailableEthCell = React.memo(({ availableETH }) => {
```

Comment:
Duplicate. The same component is in the `BondingSection.jsx`. Maybe let's extract this component to the separate file with a good name?

Reply:
Extracted and reused.